Decred Wallet/Features/Security/SecurityCodeRequestBaseViewController.swift

```diff
@@ -50,6 +50,9 @@ class SecurityCodeRequestBaseViewController: UIViewController {
     func hideError() {
         self.isInErrorState = false
     }
+    
+    func showPassphraseError(text: String) {
```

Comment:
@C-ollins nothing seems to be done inside this function

Reply:
The function is implemented in subclasses.
https://github.com/planetdecred/dcrios/pull/749/files#diff-a03f6b93d054b97dd40df7affd98091d143090b230b94b6f916541ab3801683bR224